packages/lib-sourcify/src/lib/CheckedContract.ts

```diff
@@ -756,38 +771,31 @@ function getDiffPositions(original: string, modified: string): number[] {
   return differences;
 }
 
-// Checks if a substring exists in the bytecode at a given position
+/**
+ *   Checks the raw bytecode indeed includes the auxdata diff at the given position
+ */
 function bytecodeIncludesAuxdataDiffAt(
   bytecode: string,
-  {
-    real,
-    diff,
-    offsetStart,
-    offsetEnd,
-  }: { real: string; diff: string; offsetStart: number; offsetEnd: number },
+  auxdataDiff: AuxdataDiff,
   position: number
 ): boolean {
-  const extracted = bytecode.substr(
-    position - offsetStart,
-    offsetStart + diff.length + offsetEnd
-  );
+  const { real, diffStart } = auxdataDiff;
+  // QUESTION: Can't we just use `real` instead of offSetEnd etc.?
+  const extracted = bytecode.slice(position - diffStart, real.length);
```

Comment:
We can just use `real.length` instead of `offsetStart + diff.length + offsetEnd`